changelog.d/4420.feature

```diff
@@ -0,0 +1,13 @@
+New listener resource for the federation API "openid/userinfo" endpoint
+
+Integration managers use the OpenID userinfo endpoint in the federation API to verify that user
+OpenID access tokens are valid. If the federation resource is disabled, integration managers will not be able
+to verify the access token, causing a broken experience for users. The OpenID userinfo endpoint has now been split
+to a separate `openid` resource, which is enabled by default in newly generated configuration. It is also enabled
+automatically if the federation resource is enabled.
+
+If your homeserver runs federation enabled, this change does not require any actions.
+
+If you run a homeserver with federation disabled, we recommend adding the `openid` resource to your homeserver
+configuration in the `type: http` listener `resources` list to allow your users access to
+integration manager features.
```

Comment:
This is an *incredibly* nice change log :+1: , but alas it needs to be just the one line :cry: 

However the changelog entry will point to this PR, so people will be able to find this info from the PR description :tada:

Reply:
it doesn't have to be one line (or if we strip whitespace, we can configure it not to). the real problem is breaking the dot-pointing, which i forget how to stop with markdown.

Reply:
Will condense to one line! :) I'm a fan of verbose changelogs, but I will live ;)

@hawkowl what do you mean by this, just out of interest?

> he real problem is breaking the dot-pointing, which i forget how to stop with markdown.